packages/perl.rb

```diff
@@ -35,8 +28,9 @@ def self.build
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin/"
-    system "cp cpanm #{CREW_DEST_PREFIX}/bin/"
+    FileUtils.ln_sf "#{CREW_LIB_PREFIX}/libnsl.so.1", "#{CREW_LIB_PREFIX}/libnsl.so"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin/"
```

Comment:
This line is not necessary with the line below.  `install` will create the directory.